notebooks/09_encrypt_dataset.ipynb

```diff
@@ -0,0 +1,384 @@
+      "text/plain": [
+       "<Figure size 800x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from datumaro import Visualizer\n",
+    "\n",
+    "img_id, _ = os.path.splitext(os.path.basename(img_path))\n",
+    "fig = Visualizer(dataset).vis_one_sample(img_id)\n",
+    "fig.show()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Re-export again to any dataset format with no encryption\n",
+    "\n",
+    "Because the encrypted `DatumaroBinary` format cannot easily used for you usage. In this time, we re-export it to any dataset format for the future usage. For example, COCO format is used for the export."
```

Comment:
How about 
```suggestion
    "Because the `DatumaroBinary` format is encrypted, it cannot be easily used for your purposes. In this time, we re-export it to any dataset format for the future usage. For example, COCO format is used for the export."
```